app/pb_kits/playbook/pb_time_range_inline/_time_range_inline.jsx

```diff
@@ -0,0 +1,134 @@
+/* @flow */
+
+import React from 'react'
+import classnames from 'classnames'
+import { Body, Caption, Icon } from '../'
+import DateTime from '../pb_kit/dateTime.js'
+
+type TimeRangeInlineProps = {
+  className?: String,
+  id?: String,
+  data?: String,
+  alignment?: 'left' | 'center' | 'vertical',
+  size?: 'sm' | 'xs',
+  dark?: Boolean,
+  icon?: Boolean,
+  timezone?: Boolean,
+  startTime: String,
+  endTime: String
+}
+
+const timezoneString = (dateValue) => {
+  const date = new DateTime({ value: dateValue })
+  return `${date.convertToTimezone()}`
+}
+
+const dateTimestamp = (dateValue) => {
+  const date = new DateTime({ value: dateValue })
+  return `${date.toHour()}:${date.toMinute()}${date.toMeridian()}`
+}
+
+const dateTimeIso = (dateValue) => {
+  const date = new DateTime({ value: dateValue })
+  return date.toIso()
+}
+
+const TimeRangeInline = ({
+  className,
+  alignment = 'left',
+  size = 'sm',
+  dark = false,
+  icon = false,
+  timezone = false,
+  startTime,
+  endTime,
+}: TimeRangeInlineProps) => {
+  const separator = `${'—'}`
+
+  const iconContent = () => {
+    return (
+      <If condition={icon}>
+        <Icon
+            className="pb_time_range_inline_icon"
+            dark={dark}
+            fixedWidth
+            icon="clock"
+            size={size}
+            tag="span"
+        />
+      </If>
+    )
+  }
+
+  return (
+    <div className={classnames('pb_time_range_inline_kit_' + alignment, className)}>
+      <div className="pb_time_range_inline_wrapper">
+        <If condition={size == 'xs'}>
```

Comment:
I wasn't familiar with the `<If>` component prior to this PR, but it seems like it could be a handy tool for some use cases. Is this component part of React proper, or part of another library?

Reply:
@gregblake this is [`jsx-control-statements`](https://www.npmjs.com/package/jsx-control-statements) which has been in PB for a while now and is actually part of `nitro-react` as well!